src/index.js

```diff
@@ -104,6 +104,19 @@ async function getKinesisValue(key, awsParameters) {
   return result.StreamDescription
 }
 
+/**
+ * @param key the name of the DynamoDb table to resolve
+ * @param awsParameters parameters to pass to the AWS.DynamoDB constructor
+ * @returns {Promise<AWS.DynamoDB.Table>}
+ * @see http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB.html#describeTable-property
+ */
+async function getDynamoDbValue(key, awsParameters) {
+  winston.debug(`Resolving DynamoDB stream with name ${key}`)
```

Comment:
@jacob-meacham I've doubted between dynamo or dynamoDB. Don't hesitate to revert to me if i should change it again.